ctapipe_io_magic/tests/test_magic_event_source.py

```diff
@@ -13,6 +13,26 @@
     test_calibrated_real_dir / "20210314_M2_05095172.002_Y_CrabNebula-W0.40+035.root",
 ]
 
+test_calibrated_real_only_events = [
+    test_calibrated_real_dir
+    / "20210314_M1_05095173.001_Y_CrabNebula-W0.40+035_only_events.root",
```

Comment:
if you want to make the code a bit shorter, you can store here only the file name, and add the path below in a loop when creating ```test_calibrated_missing_trees```
```
test_calibrated_real_only_events =  "20210314_M1_05095173.001_Y_CrabNebula-W0.40+035_only_events.root"
test_calibrated_real_only_drive =  "20210314_M1_05095173.001_Y_CrabNebula-W0.40+035_only_drive.root"
...
test_calibrated_missing_trees = ([
    test_calibrated_real_dir / x
    for x in [test_calibrated_real_only_events, test_calibrated_real_only_drive, ...]
])

Reply:
This I cannot do because some tests rely on the single files, so I need the full path when defining them. In any case, it does not impact the code so much (at least for the moment!).